public/javascripts/hosts.js

```diff
@@ -25,6 +25,7 @@ function UpdateMonitorPage(){
       $(`#${h}_mem_used`).html((data['virtual_memory']['percent']).toFixed(1)+"%").css("color", data['virtual_memory']['percent'] > 75 ? 'red' : 'black');
       $(`#${h}_swap`).html((data['swap_memory']['percent']).toFixed(1) + "%").css("color", data['swap_memory']['percent'] > 75 ? 'red' : 'black');
       $(`#${h}_load`).html(data['load'].toString());
+      $(`#${h}_drift`).html(data['drift'].toFixed(1) + ' s');
```

Comment:
```suggestion
      $(`#${h}_drift`).html(data['drift'].toFixed(1) + ' s').css("color", Math.abs(data['drift']) > 2 ? 'red' : 'black');
```
Not sure how java does/needs imports